src/material-experimental/mdc-form-field/directives/floating-label.ts

```diff
@@ -6,40 +6,41 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Directive, ElementRef, Input, OnDestroy} from '@angular/core';
-import {MDCFloatingLabel} from '@material/floating-label';
+import {Directive, ElementRef, Input} from '@angular/core';
+import {ponyfill} from '@material/dom';
 
 /**
- * Internal directive that creates an instance of the MDC floating label
- * component. Using a directive allows us to conditionally render a floating label
- * in the template without having to manually instantiate the `MDCFloatingLabel` component.
+ * Internal directive that maintains a MDC floating label. The directive does not use
+ * the foundation or component MDC provides for the floating label as those are not
```

Comment:
I don't think we use _any_ of the MDC components, just the foundations, so that doesn't need to be explicitly called out.

Also `not needed (in terms of bundle size)` sounds a little weird. I would maybe say:
```
This directive does not use the `MDCFloatingLabelFoundation`, as it is not worth the
size cost of including it just to measure the label and toggle some classes
```

Reply:
@mmalerba I called it out explicitly here because the floating label (before this change) actually used the component (as it was sufficient to rely on the default adapter).

Though, I agree that this is not really necessary as in reality the logic originated from the foundation. I'll update it to what you proposed. Thanks!